weave/trace_server/trace_server_interface.py

```diff
@@ -253,6 +288,7 @@ class CallsQueryReq(BaseModel):
     # Sort by multiple fields
     sort_by: typing.Optional[typing.List[_SortBy]] = None
     query: typing.Optional[Query] = None
+    add_cost: typing.Optional[bool] = False
```

Comment:
can you add a unit test that exercises this path - going through the client?

Reply:
when I go through the client it hits the sqlite server rather than clickhouse, Is there something that I need to enable to correctly test through client?

Reply:
every test that uses the `client` fixture will be run twice in CI - once for SQLite and once for CH. To force CH, you can run pytest with `--weave_server=clickhouse`. Checkout the `client` fixture for details